wikipendium/wiki/models.py

```diff
@@ -135,6 +135,8 @@ def clean(self):
         if '/' in self.title:
             raise ValidationError('Title cannot contain slashes')
 
+        self.content.replace('\r\n', '\n').replace('\r', '\n')
```

Comment:
What I'm saying is that calling .replace on a string variable won't change the variable, but return a new string. So we need `self.content = self.content.replace('\r\n', '\n').replace('\r', '\n')` for it to work as intended.

PS: A test would be useful